sbt-dump-event-schema/src/test/resources/target.yaml

```diff
@@ -0,0 +1,65 @@
+- name: org.random.project.Data
+  typeSymbol: trait
+- name: org.random.project.Data.AdditionalData
+  fields:
+  - name: a
+    typeName: scala.Int
+  typeSymbol: class
+- parents:
+  - org.random.project.Data
+  name: org.random.project.Data.ClassTest
+  fields:
+  - name: a
+    typeName: java.lang.String
+  - name: b
+    typeName: scala.Int
+  - name: c
+    typeName: scala.Double
+  typeSymbol: class
+- parents:
+  - org.random.project.Data
+  name: org.random.project.Data.ClassWithAdditionData
+  fields:
+  - name: ad
+    typeName: org.random.project.Data.AdditionalData
+  typeSymbol: class
+- parents:
+  - org.random.project.Data
+  name: org.random.project.Data.DeepTrait
+  typeSymbol: trait
+- parents:
+  - org.random.project.Data
+  name: org.random.project.Data.DeeperTrait
+  typeSymbol: trait
+- parents:
+  - org.random.project.Data.DeeperTrait
+  name: org.random.project.Data.DeepestClass
+  fields:
+  - name: a
+    typeName: scala.Int
+  typeSymbol: class
+- parents:
+  - org.random.project.Data
+  name: org.random.project.Data.GenericsTest
+  fields:
+  - name: a
+    typeName: scala.collection.immutable.Map[String,Int]
+  - name: b
+    typeName: scala.Option[Int]
+  - name: c
+    typeName: scala.Tuple3[Int,Int,Int]
+  typeSymbol: class
+- name: org.random.project.Data.TraitWithAnnotation
+  parents:
```

Comment:
Whooops looks like the order isn't deterministic, comparing this entry and the one above :/